### PR TITLE
SAK-32315 (Bug fix) : Problem uploading file to Urkund with multipart…

### PR DESCRIPTION
**Pull request by frasese**
… post http request

+ Refactor : Included UrkundSubmission object to better manage
parameters

### CODE REVIEW

File: content-review/impl/urkund/src/main/java/org/sakaiproject/contentreview/urkund/util/UrkundAPIUtil.java [hunk 1]
```diff
@@ -58,32 +56,33 @@ public static String postDocument(String baseUrl, String submitterEmail, String
...
+				httppost.addHeader("x-urkund-submitter", submission.getSubmitterEmail());
+				httppost.addHeader("x-urkund-anonymous", Boolean.toString(submission.isAnon()));
+				httppost.addHeader("x-urkund-subject", submission.getSubject());
+				httppost.addHeader("x-urkund-message", submission.getMessage());
 				//------------------------------------------------------------
 
 				HttpResponse response = httpClient.execute(httppost);
 				HttpEntity resEntity = response.getEntity();
-
+
```
Comment (ottenhoff) [inline content-review/impl/urkund/src/main/java/org/sakaiproject/contentreview/urkund/util/UrkundAPIUtil.java]: extra tab

File: content-review/impl/urkund/src/main/java/org/sakaiproject/contentreview/urkund/util/UrkundAPIUtil.java [hunk 2]
```diff
@@ -58,32 +56,33 @@ public static String postDocument(String baseUrl, String submitterEmail, String
...
 				HttpResponse response = httpClient.execute(httppost);
 				HttpEntity resEntity = response.getEntity();
-
+	
 				if (resEntity != null) {
 					ret = EntityUtils.toString(resEntity);
 					EntityUtils.consume(resEntity);
 				}
-
+
```
Comment (ottenhoff) [inline content-review/impl/urkund/src/main/java/org/sakaiproject/contentreview/urkund/util/UrkundAPIUtil.java]: extra tab

File: content-review/impl/urkund/src/main/java/org/sakaiproject/contentreview/urkund/UrkundSubmission.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/**********************************************************************************
+ *
+ * Copyright (c) 2017 Sakai Foundation
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.osedu.org/licenses/ECL-2.0
```
Comment (ottenhoff) [inline content-review/impl/urkund/src/main/java/org/sakaiproject/contentreview/urkund/UrkundSubmission.java]: this link is very old ... 

https://opensource.org/licenses/ECL-2.0

**Comment by mpellicer** [comment]
Please squash it in one commit!

**Pull request merged**